docs/blog/2022-12-12-Building-OKD-payload.md

```diff
@@ -0,0 +1,265 @@
+
+> The release-controller automatically builds OpenShift release images when new images are created for a given OpenShift release. It detects changes to an image stream, launches a job to build and push the release payload image using `oc adm release new`, and then runs zero or more ProwJobs against the artifacts generated by the payload. 
+
+A release image is nothing more than a ClusterVersionOperator image (CVO), with an extra layer containing the `release-manifests` folder. This folder contains :
+* `image-references`: a list of all known images with their SHA digest,
+* yaml manifest files for each operator controlled by the CVO.
+
+The list of images that is included in the `release-manifests` is calculated from the `release` [image stream](https://developers.redhat.com/blog/2019/09/20/using-red-hat-openshift-image-streams-with-kubernetes-deployments#), taking :
+* all images with label `io.openshift.release.operator=true` in that image stream
+* plus any images referenced in the `/manifests/image-references` file within each of the images with this label.
+
+As you can imagine, the list of images in a release can change from one release to the next, depending on:
+* new operators being delivered within the OpenShift release
+* existing operators adding or removing an operand image
+* operators previously included that are removed from the payload to be delivered independently, through OLM instead.
+
+In order to list the images contained in a release payload, run this command:
+
+```shell
+oc adm release info ${RELEASE_IMAGE_URL}
+``` 
+
+For example:
+
+```shell
+oc adm release info quay.io/okd/scos-release:4.12.0-0.okd-scos-2022-12-02-083740 
+```
+
+Now that we've established what needs to be built, let's take a deeper look at how the OKD on SCOS payload is built.
+
+
+## Building OKD/SCOS the Prow way :railway_track: 
+
+The obvious way to build OKD on SCOS is to use [Prow](https://docs.prow.k8s.io/docs/) - THE Kubernetes-based CI/CD system, which is what builds OCP and OKD on FCOS already today. This is what Kubernetes uses upstream as well. :shrug:
+
+For a new OKD release to land in the [releases](https://origin-release.ci.openshift.org) page, there's a whole bunch of Prow jobs that run. Hang on! It's a long story...
+
+
+### ImageStreams
+
+Let's start by the end :wink:, and prepare a new image stream for OKD on SCOS images. This ImageStream (IS) is a placeholder for all images that form the OKD/SCOS payload.
+
+For OKD on Fedora CoreOS (OKD/FCOS) it's named `okd`.For OKD/SCOS, this IS is named `okd-scos`.  
+
+This IS includes all payload images contained in the specific `OKD` release based on CentOS Stream CoreOS (SCOS)  
+
+Among these payload images, we distiguish:
+* Images that can be shared between OCP and OKD. These are built in Prow and mirrored into the `okd-scos` IS.  
```

Comment:
I think `IS` is not a great shortcut, it looks like uppercased "is", so may confuse some users.